templates/var/lib/ansible/playbooks/main.yml

```diff
@@ -38,6 +38,8 @@
   - name: Restart node service
     service: name={{openshift.common.service_type}}-node state=restarted
 
+- include: /var/lib/ansible/playbooks/ipfailover.yml
+
```

Comment:
Would it make sense (if possible) to make ipfailover setup only when neutron lbaas is used? IOW move router_vip resources into loadbalancer_neutron.yaml and run this playbook only when loadbalancer_type==neutron?